plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -45,19 +45,18 @@
     
     <library
           id="objectify"
-          group="appengine"
+          group="appengine,flexible"
```

Comment:
Maybe we should just promote Datastore Cloud client library on flex?

https://cloud.google.com/appengine/docs/flexible/java/migrating#datastore

The doc also says (might be out of date),

> At this time ORM libraries that available in the standard environment such as ndb and Objectify are not supported outside of the standard environment.

Reply:
That doc is out of date. 6.0 supports flex: "Objectify v5 and prior use the Google App Engine SDK and therefore can only be used within Google App Engine Standard. Objectify v6+ uses the new Google Cloud SDK and can be used from anywhere - GAE Standard, GAE Flex, GCE, or outside Google Cloud entirely."